frontend/ui/src/pages/Channels/ConnectedChannelsList/ChannelListItem.tsx

```diff
@@ -27,32 +24,12 @@ const ChannelItem = (props: ChannelItemProps) => {
   const {channel} = props;
   const [deletePopupVisible, setDeletePopupVisible] = useState(false);
 
-  const disconnectChannelRequestPayload = {
-    channelId: channel.id,
-  };
-
-  const getSourceLogo = (channel: Channel) => {
-    switch (channel.source) {
-      case ChannelSource.facebook:
-        return <FacebookLogo />;
-      case ChannelSource.google:
-        return <GoogleLogo />;
-      case ChannelSource.twilioSMS:
-        return <SMSLogo />;
-      case ChannelSource.twilioWhatsapp:
-        return <WhatsappLogo />;
-      case ChannelSource.chatPlugin:
-        return <AiryLogo />;
-      default:
-        return <AiryLogo />;
-    }
+  const isPhoneNumberSource = () => {
+    return channel.source === 'twilio.sms' || channel.source === 'twilio.whatsapp';
   };
 
-  const ChannelIcon = ({channel}: {channel: Channel}) => {
```

Comment:
why did you remove this? this Component is returning the default logo of the source or the given image of the imageUrl (as Facebook pages)